src/components/Icon/assets/chat-bubble.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Hello @kostasdano, I suggest to remove the width and the height from the svg since we handle these properties using css.

Reply:
nice catch, thanks! :D